eksctl/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM alpine:edge as downloads
+WORKDIR /downloads
+
+ARG EKSCTL_VERSION=0.11.1
+ARG KUBECTL_VERSION=1.15.6
```

Comment:
@cw-sakamoto 
Does the version match the kubernetes cluster created by EKS?
https://docs.aws.amazon.com/eks/latest/userguide/platform-versions.html

Reply:
@cw-ozaki 
kubectl supports one version before and after, so we have increased it by one from the current EKS version in preparation for the next version.
FLY: https://kubernetes.io/docs/setup/release/version-skew-policy/#kubectl